CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ Bug fixes:
 - [Models] PyTorch models can now be saved with code dependencies, allowing model classes to be loaded successfully in new environments (#842, #836, @dbczumar)
 - [Artifacts] Fixed a timeout when logging zero-length files to DBFS artifact stores (#818, @smurching)
 
+Small bug fixes and docs updates (#845, @stbof; #840, @grahamhealy20; #839, @wilderrodrigues)
```

Comment:
Aren't these all documentation fixes?
#846 also pinned some py libraries. We don't need to call that out -- but that was intended rite?

Reply:
Good point! Removed `bug fixes`. I didn't end up cherry-picking 846; I've pinned the dependency in the `test-requirements.txt` file on the release branch.